<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExperimentChooserDialog</class>
 <widget class="QDialog" name="ExperimentChooserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Experiment Launcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/experiment.png</normaloff>:/icons/icons/experiment.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Patient's Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lePatientName"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Patient's Age</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lePatientAge"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leEmail"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Experiment</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cboxExperiments">
     <item>
      <property name="text">
       <string>Sequence 1:  Reading &gt; Binding UC &gt; Binding BC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Reading</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Binding UC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Binding BC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fielding</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExperimentChooserDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExperimentChooserDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
